add versioning info into the data connector docs (#381)
transposit/docs: references/create-a-data-connector.md

Commit your changes and push back to Transposit. Transposit will run some validation on your OpenAPI file.

`git push origin master`

> You can also tag your API using [SemVer](https://semver.org/) formatting. Applications that use your connector can then control which verison they are using. This is recommended.

Reviewer: ```suggestion
You can also tag your API using [SemVer](https://semver.org/) formatting. Tags are recommended so that applications using your data connector can control which version they are using.
```
Or something about how you won't break them if you update the connector? Or is that implied?
